reactioncommerce/reaction-next-starterkit
# feat: PDP basic info

Issue: #22 ( depends on #21 )
Impact: **minor**
Type: **feat**

## Issue

Create basic info on the PDP page with price, vendor, and description fields.

## Solution 

Create a component that can

- Display price, vendor, description if the data is passed to the component.

## Test

- Start the app and go to `http://localhost:4000/product/1234`
- See the following

![reaction_shop](https://user-images.githubusercontent.com/42217/39067804-d29d03f2-448e-11e8-8cd0-6cf294a47b3c.png)
